### PR TITLE
support expiry date for facebook api >= 2.3

### PR DESCRIPTION
**Pull request by ixna**
I'm troubled to get the expiry datetime as i use newer version of facebook API version. Please check my commits. Thanks.

### CODE REVIEW

File: tests/test_utils.py [hunk 1]
```diff
@@ -61,7 +61,7 @@ def test_get_extended_access_token():
 def test_get_extended_access_token_v23_plus():
     mock_request.return_value.status_code = 200
     mock_request.return_value.content = (
-        '{"access_token":"<extended access token>","token_type":"bearer"}'
+        '{"access_token":"<extended access token>","token_type":"bearer", "expires_in":5170982}'
```
Comment (jgorset) [inline tests/test_utils.py]: I may be missing something, but I'm not getting a key called `expires_in` from this endpoint in Graph API versions 2.3 and above:

```http
GET /v2.3/oauth/access_token?grant_type=fb_exchange_token&client_id=xxx&client_secret=xxx&fb_exchange_token=xxx HTTP/1.1
Host: graph.facebook.com
Connection: close


HTTP/1.1 200 OK
Access-Control-Allow-Origin: *
Pragma: no-cache
Cache-Control: private, no-cache, no-store, must-revalidate
facebook-api-version: v2.3
Expires: Sat, 01 Jan 2000 00:00:00 GMT
Content-Type: application/json; charset=UTF-8
x-fb-trace-id: EdS3FRiHsz8
x-fb-rev: 2728374
Vary: Accept-Encoding
X-FB-Debug: AQi1j55HPsiRM/ClUEaoHh1vXLdi6knogi/0Dm4XPVQn5ofy543pXk0uEMNrYufp1cr3jsyvJp/QYyO/8phW/A==
Date: Thu, 08 Dec 2016 14:24:38 GMT
Connection: close

{"access_token":"EAAG6WgWqL5cBAIuOwC2TPCJzA3jEypMKvNwu0ZBPZAwZA0oZB91znZBCAJjZBtGlorVpSQWVj9sb2oG7OwDIYX8h6wuIbFLcfpZCnInW73KeOjqcZAx4K0QL4RyZBcX2twgZBZCMEPJZCzy4TGDbLP8YT6Dr","token_type":"bearer"}
```

Do you know why you're getting it, @irkho?

File: tests/test_utils.py [hunk 2]
```diff
@@ -86,7 +86,9 @@ def test_get_extended_access_token_v23_plus():
     )
 
     assert_equal(access_token, '<extended access token>')
-    assert not expires_at
+    # there is data for expiration check on key 'expires_in', 
+    # so i comment this one
+    #assert not expires_at
```
Comment (jgorset) [inline tests/test_utils.py]: We should assert that it contains what we expect it to contain, then. ;-)

**Comment by jgorset** [comment]
Sorry for not getting around to this one, @irkho! I left some comments.

File: tests/test_utils.py (hunk 1 above)
Comment (Badzyo) [inline tests/test_utils.py]: @jgorset The latest API version is 2.8, and it returns a key called `expires_in`.
```bash
curl -D - "https://graph.facebook.com/v2.8/oauth/access_token?client_id=xxx&grant_type=fb_exchange_token&client_secret=xxx&fb_exchange_token=xxx"

HTTP/1.1 200 OK
Access-Control-Allow-Origin: *
Pragma: no-cache
Cache-Control: private, no-cache, no-store, must-revalidate
facebook-api-version: v2.8
Expires: Sat, 01 Jan 2000 00:00:00 GMT
Content-Type: application/json; charset=UTF-8
x-fb-trace-id: CXWNY9XCDq/
x-fb-rev: 2728374
Vary: Accept-Encoding
X-FB-Debug: a/gcxxCFLErpXmls/SXNt0Dr0VchY3wik+ivDZ4CbhX6EvTaPEuBKcaj83gMQXCiE30g+BeUTDYglFXQ9Si3yA==
Date: Thu, 08 Dec 2016 14:44:53 GMT
Transfer-Encoding: chunked
Connection: keep-alive

{"access_token":"xxx","token_type":"bearer","expires_in":5103006}
```

Comment (jgorset) [inline tests/test_utils.py]: Mine doesn't, @Badzyo:

```http
GET /v2.8/oauth/access_token?grant_type=fb_exchange_token&client_id=xxx&client_secret=xxx&fb_exchange_token=xxx HTTP/1.1
Host: graph.facebook.com
Connection: close
User-Agent: Paw/3.0.13 (Macintosh; OS X/10.12.1) GCDHTTPRequest

{
  "access_token": "xxx",
  "token_type": "bearer"
}
```

Comment (irkho) [inline tests/test_utils.py]: Hello @jgorset..Strange 😅 , mine also returned with `expires_in` key, the same as @Badzyo 's, maybe there is something to do with region or app configuration. Let me try to investigate more.

**Comment by mgalgs** [comment]
I just encountered this issue.  I was calling `get_extended_access_token` without specifying any version and my just started breaking today.  I was no longer getting expiration timestamps.  This PR fixed the issue for me (and I've learned my lesson about not specifying a version ;) ).

**Comment by jgorset** [comment]
Well, this must be the longest it's ever taken to get a pull request merged. 😅 Sorry guys, and thanks for this. :sparkles: :cake: :sparkles:

**Comment by jgorset** [comment]
This is in v1.0.11 on PyPi now.

**Pull request merged**